webcompat/api/endpoints.py

```diff
@@ -187,14 +187,21 @@ def get_category_from_search(issue_category):
     category_list = ['contactready', 'needscontact',
                      'needsdiagnosis', 'sitewait']
     params = request.args.copy()
+    query_string = ''
+
+    # restrict results to our repo.
+    query_string += " repo:{0} ".format(REPO_PATH)
```

Comment:
Actually, do we need this? All the possible return values of this method call `get_search_results` which does the exact same thing, so are we sending a double `repo:foo`?